phpunit.xml.dist

```diff
@@ -32,11 +32,11 @@
 
     <php>
         <env name="LIGHTHOUSE_TEST_DB_DATABASE" value="test"/>
-        <env name="LIGHTHOUSE_TEST_DB_HOST" value="mysql"/>
+        <env name="LIGHTHOUSE_TEST_DB_HOST" value="localhost"/>
```

Comment:
You can override this through your own `phpunit.xml`. The existing config works within the included docker-compose stack.

Reply:
Ups. That was not supposed to be commited. Nice catch. 😃